docs/content/preview/migrate/manual-import/_index.md

```diff
@@ -17,11 +17,18 @@ type: indexpage
 
 The steps below cover how to manually migrate PostgreSQL data and applications to YugabyteDB.
 
+- [Convert a PostgreSQL schema](migrate-schema/)
+- [Migrate a PostgreSQL application](migrate-application/)
+- [Export PostgreSQL data](export-data/)
+- [Prepare a cluster](prepare-cluster/)
+- [Import PostgreSQL data](import-data/)
+- [Verify a migration](verify-migration/)
+
 {{< tip title="Migrate using YugabyteDB Voyager" >}}
```

Comment:
It wouldn't hurt to put this tip on the import page (and maybe the export page if it makes sense?)...